subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3273,14 +3273,26 @@ static void le_set_ext_adv_param(struct net_buf *buf, struct net_buf **evt)
 		return;
 	}
 
+	min_interval = sys_get_le24(cmd->prim_min_interval);
+
+	if (IS_ENABLED(CONFIG_BT_CTLR_PARAM_CHECK)) {
+		uint32_t max_interval = sys_get_le24(cmd->prim_max_interval);
+
+		if ((min_interval > max_interval) ||
+		    (min_interval < 0x0020) ||
+		    (max_interval > 0x4000)) {
```

Comment:
```suggestion
		    (min_interval < BT_LE_ADV_INTERVAL_MIN) ||
		    (max_interval > BT_LE_ADV_INTERVAL_MAX)) {
```

The `BT_LE_ADV_INTERVAL_MAX` is defined as `0x4000`, but the spec states: 
![image](https://user-images.githubusercontent.com/2520413/165913634-648594b2-9fae-4001-b1bf-695271cb76dc.png)

Shouldn't it be `0xFFFFFF`, or am I missing something?